enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -89,6 +91,12 @@ class Meta:
         read_only_fields = ['uuid']
         extra_kwargs = {
             'uuid': {'read_only': True},
+            'display_name': {
+                'allow_blank': False,
```

Comment:
[clarification] If this field has `'allow_blank': False` initially, does this mean the provisioning tool will not work correctly until any corresponding changes are made on the provisioning tool? If that's the case, should we allow blanks on the API side of things for now until the provisioning tool is passing along `display_name` in its payload?